.github/workflows/post_peek_screenshot.yml

```diff
@@ -54,7 +54,7 @@ jobs:
             I'm Devicons' Peek Bot and I just peeked at the icons that you wanted to add using [icomoon.io](https://icomoon.io/app/#/select).
             Here is the result below:
 
-            ![Peeked Icons (top left)]({0})
+            {0}
 
             Here are the zoomed-in screenshots of the added icons:
```

Comment:
When you add more than two icons for the font versions, it adds up an `\n` in the GitHub comment:

![image](https://user-images.githubusercontent.com/51391473/108760642-87511b00-7513-11eb-8fb4-02e7d735d1ee.png)

Reply:
Has this been fixed yet? Or is it fixed in this PR? 😄

Reply:
@Panquesito7 a few seconds too late before. I will create a new issue based on your report! Thanks! 👍🏻